packages/lisk-p2p/src/p2p.ts

```diff
@@ -228,6 +231,10 @@ export class P2P extends EventEmitter {
 		this._discoveryInterval = config.discoveryInterval
 			? config.discoveryInterval
 			: DEFAULT_DISCOVERY_INTERVAL;
+
+		this._peerHandshakeChecks = config.peerhandShakeChecks
```

Comment:
Inconsistent capitalization: `peerhandShakeChecks` should be `peerHandshakeChecks`.

I prefer `peerHandshakeCheck` (singular) but this is minor.